browser/js/cart/cart.js

```diff
@@ -25,7 +25,7 @@ app.controller('CartCtrl', function ($scope, $http, cartInfo, cartFactory) {
 		var filmId = film._id;
 		$http.put('/api/cart', {filmId: filmId}).
 		    success(function(cartInfo) {
-		    	$scope.allFilmsInCart = cartInfo.films;
+		    	$scope.allLineItemsInCart = cartInfo.lineItems;
```

Comment:
Not sure if issue is here... but can no longer use "add to cart" button in pdp
![image](https://cloud.githubusercontent.com/assets/8867156/7424445/06c70dbc-ef6f-11e4-981b-13d72a481ec1.png)

How do I get my When Harry Met Sally? 
![image](https://cloud.githubusercontent.com/assets/8867156/7424451/19878d0a-ef6f-11e4-8f84-5e6c283bf9e1.png)